frontend/js/src/utils/SearchTrackOrMBID.tsx

```diff
@@ -185,22 +211,35 @@ export default function SearchTrackOrMBID({
           </button>
         </span>
         {Boolean(searchResults?.length) && (
-          <div className="track-search-dropdown">
-            {searchResults.map((track) => {
-              return (
-                <button
-                  key={track.recording_mbid}
-                  type="button"
-                  onClick={() => {
-                    selectSearchResult(track);
-                    reset();
-                  }}
-                >
-                  {`${track.recording_name} - ${track.artist_credit_name}`}
-                </button>
+          <select
+            className="track-search-dropdown"
+            size={Math.min(searchResults.length, 5)}
```

Comment:
Let's be a bit more generous with the height of this dropdown
```suggestion
            size={Math.min(searchResults.length, 8)}
```